riscv/insns/lpad.h

```diff
@@ -0,0 +1,9 @@
+if (p->extension_enabled(EXT_ZICFILP)) {
+  if (ZICFILP_READ_XLPE(STATE.v, STATE.prv) && STATE.elp == elp_t::LP_EXPECTED) {
+    software_check(pc % 4 == 0 &&
+                   ((READ_REG(7) >> 12) == insn.u_imm() || insn.u_imm() == 0),
```

Comment:
I think a plain `7` here is acceptable, since in the Zicfilp spec, `x7` is referred to directly without additional abstractions, so keeping the `7` plain here does not really constitute a magic number use case, which everyone wants to avoid due to confusion it may cause, and moreover, this approach provides a clearer connection to the wording used in the spec.

What do you say?

Reply:
My suggestion is that you should define something like `X7` instead of directly writing index.